skimage/restoration/_denoise.py

```diff
@@ -222,8 +222,6 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
     # if image.max() is 0, then dist_scale can have an unverified value
     # and color_lut[<int>(dist * dist_scale)] may cause a segmentation fault
     # so we verify we have a positive image and that the max is not 0.0.
-    if min_value < 0.0:
-        raise ValueError("Image must contain only positive values")
 
     if max_value == 0.0:
         raise ValueError("The maximum value found in the image was 0.")
```

Comment:
```suggestion
```
This is no more relevant if we want to support negative values :wink:

Reply:
Thanks